### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by afspicciati**
@Mick-Ross

### CODE REVIEW

File: Lab1.ipynb [hunk 1]
```diff
@@ -29,13 +29,20 @@
...
+    "#### A.\n",
+    "The normal distribution is a symmetrically distributed bell curve of the outcomes or potential outcomes for a given dataset. The probability density of the normal distribution is given by:\n",
+    "\n",
+    "$$P(x) = \\frac{1}{\\sigma\\sqrt{2pi}} e^{-(x-\\mu)^2/({2\\sigma^2})}$$\n",
+    "\n",
+    "where x=value, $\\mu$ = mean (the center of the distribution), and $\\sigma^2$ is the variance of the distribution. For the standard normal distribution (where mean=0 and sigma=1), the Z-score is the value for which a given percentage of the data is above that value. \n",
+    "\n",
     "#### B.\n",
-    "erfc(1) tells us that 15.7% of distribution is above Z score of 1 while erf(1) tells us 84.2% of data is below Z score of 1. "
+    "erfc(1) tells us that 15.7% of distribution is above Z score of 1 while erf(1) tells us 84.3% of data is below Z score of 1. These values match up with those from the Z table. I also show that erfc(1)+erf(1) = 1, to demonstrate that the integral of the normal distribution across all space is 1. I have some confusion why erf(1) and cdf(1) are not equivalent in the 1/100th place, as these should calculate the same quantity."
```
Comment (Mick-Ross) [inline Lab1.ipynb:40]: Here you explained the erfc() function well, but you could also explain cdf() as well to fully flesh out your explanation of your code here.

File: Lab1.ipynb [hunk 2]
```diff
@@ -289,66 +358,197 @@
...
+    "## Non-continuous distributions (Poisson)\n",
+    "\n",
+    "#### A.\n",
+    "The Poisson distribution is a model for when the average time between events is known, but when the events will occur is not known. For a given number of events k, the probability of measuring k events in a given timeframe is:\n",
+    "\n",
+    "$$Pr(x=k) = \\frac{\\lambda^k e^{-\\lambda}}{k!} $$ \n",
+    "\n",
+    "where $\\lambda$ is the expected value of x, and its variance. This distribution is discrete, because you cannot measure a fractional number of events taking place.\n",
+    "\n",
+    "#### B."
```
Comment (Mick-Ross) [inline Lab1.ipynb:428]: For this section, you did a great job of creating plots of different parameters, but you should also explain the features of each plot in words. How does changing the parameters of your Poisson distribution affect its shape?

File: Lab1.ipynb [hunk 3]
```diff
@@ -289,66 +358,197 @@
...
+   "id": "d13542e5-17fc-4534-8669-4175ad4d08c0",
+   "metadata": {},
+   "source": [
+    "#### D.\n",
+    "\n",
+    "Since probabilities and sigmas are discrete, this means that there will be a certain value measurement that crosses the 5-Sigma barrier, while 1 less than that value would not. For experiments this would create a sort of 'benchmark' value that you are hoping to produce. \n",
+    "\n",
+    "#### E.\n",
+    "\n",
+    "It makes sense that discrete values can still have non-discrete means. For example, the average value of a die roll would be 3.5. So the expected value of a measurement can change continuously with changes in the parameters, but the measured value will still always be discrete."
```
Comment (Mick-Ross) [inline Lab1.ipynb:545]: Could you think of any other parameters that changes continuously, and what doesn't? Other than just the mean, probabilities, and sigmas? I also struggled to think of other measurements to talk about.

File: Lab1.ipynb [hunk 4]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (Mick-Ross) [inline Lab1.ipynb:2]: I'm struggling to wade through this raw jupyter file, so bear with me. I'll just be commenting on the content such as explanations and figures instead of all the metadata and source code in this JSON file.

File: Lab1.ipynb (hunk 1 above)
Comment (Mick-Ross) [inline Lab1.ipynb]: Here you explained the erfc() function well, but you could also explain cdf() as well to fully flesh out your explanation of your code here.

File: Lab1.ipynb (hunk 2 above)
Comment (Mick-Ross) [inline Lab1.ipynb:438]: For this section, you did a great job of creating plots of different parameters, but you should also explain the features of each plot in words. How does changing the parameters of your Poisson distribution affect its shape?

File: Lab1.ipynb (hunk 3 above)
Comment (Mick-Ross) [inline Lab1.ipynb]: Could you think of any other parameters that changes continuously, and what doesn't? Other than just the mean, probabilities, and sigmas? I also struggled to think of other measurements to talk about.

File: Lab1.ipynb [hunk 5]
```diff
@@ -289,66 +358,197 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d13542e5-17fc-4534-8669-4175ad4d08c0",
+   "metadata": {},
+   "source": [
+    "#### D.\n",
+    "\n",
+    "Since probabilities and sigmas are discrete, this means that there will be a certain value measurement that crosses the 5-Sigma barrier, while 1 less than that value would not. For experiments this would create a sort of 'benchmark' value that you are hoping to produce. \n",
```
Comment (Mick-Ross) [inline Lab1.ipynb:554]: Here you already run with the claim that probabilities and sigmas are discrete, but I think for the sake of this lab you'd also want to explain why that is.

**Review by Mick-Ross — COMMENTED** [review]
Overall, great work!
Just need to patch up some holes in your explanations. But for what you have, you were clear and concise, and never too wordy.
I think wording out your knowledge of statistics more will improve the communication of this lab for a reader.

**Comment by afspicciati** [comment]
@hrzhao76 , here is the completed Lab1 with revisions after the peer review!
